l5kit/l5kit/environment/test_gym.py

```diff
@@ -0,0 +1,27 @@
+import os
```

Comment:
This file should go to the examples folder if it is an example, otherwise should be an integration test.